MidasCivil_Adapter/CRUD/Create/Elements/Element.cs

```diff
@@ -72,9 +72,11 @@ private bool CreateCollection(IEnumerable<FEMesh> meshes)
 
             CreateGroups(meshes);
```

Comment:
Can you move the the `CreateGroups` method until after the `Convert.FEMesh` is called - this is because we need to assign the `Face` `AdapterIds` before creating groups.